style.css

```diff
@@ -387,6 +395,54 @@ form button {
   margin: auto;
 }
 
+@media only screen and (max-width: 768px) {
+  .nav-mobile {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    gap: 46px;
+    top: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    z-index: 999;
+    height: 100vh;
+    padding-left: 30px;
+    padding-right: 43px;
+    padding-top: 24px;
+    background-color: #6070ffef;
+    animation: SlideInLeft 0.5s ease-in-out;
+  }
+
+  .nav-mobile button {
+    margin-right: 0;
+    width: 100%;
+    text-align: right;
+    font-size: 20px;
+    font-weight: 600;
+    border-style: none;
+    color: white;
+    background: none;
+  }
+
+  .nav-mobile ul {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+  }
+
+  .nav-mobile ul li {
+    list-style: none;
+  }
+
+  .a-mobile {
+    font-size: 32px;
+    font-weight: 600;
+    text-decoration: none;
+    color: white;
+  }
+}
```

Comment:
- [ ] Kindly consider adding a backdrop filter for the mobile menu to match your design with the Figma design 

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/223158322-0ec8ee0e-fe73-4857-946b-fef1acec5701.png)  | ![image](https://user-images.githubusercontent.com/53402600/223158195-6122d234-d4da-4da1-b299-fd4584c5521c.png) |

Reply:
Hello @omarsalem7,
okay, thank you for your review.
the required change is taken into consideration.